ha/monitor/k8s/object_monitor.py

```diff
@@ -167,3 +172,50 @@ def run(self):
                 break
         Log.info(f"Stopping the {self.name}...")
 
+    def _is_published_alert(self, alert) -> bool:
+        """
+        Check incoming alert is already published or not.
+        If incoming alert is not found in published alerts, then
+        it is a new alert to publish. Alert will be stored like,
+        self._published_alerts = {
+            resource_name: { generation_id : alert }
+            }
+
+        Returns:
+            True if it is published already
+            False if it is a new alert
+        """
+        incoming_alert = alert.to_dict().copy()
```

Comment:
Please rebase. alert is not getting used now. You can use Event class which is in fault tolerance right now. Will be moved to utils

Reply:
Done.

Reply:
Thanks!